coprocess_grpc.go

```diff
@@ -85,13 +85,29 @@ func (d *GRPCDispatcher) Reload() {}
 // HandleMiddlewareCache isn't used by gRPC.
 func (d *GRPCDispatcher) HandleMiddlewareCache(b *apidef.BundleManifest, basePath string) {}
 
+func grpcCallOpts() grpc.DialOption {
+	recvSize := config.Global().CoProcessOptions.GRPCRecvMaxSize
+	sendSize := config.Global().CoProcessOptions.GRPCSendMaxSize
+	if recvSize > 0 && sendSize > 0 {
```

Comment:
use case, override default SendSize without having to also set Receive Size?

Reply:
Added this